react/components/CreateOrganizationButton/CreateOrganizationButton.tsx

```diff
@@ -1,20 +1,64 @@
 import React, { useState } from 'react'
```

Comment:
This is the main file with the business logic of the changes. The others are either: translation or dependency-related.